ActivityAnalyses/ActivityClasses/gaitAnalysis.py

```diff
@@ -0,0 +1,381 @@
+# -*- coding: utf-8 -*-
+"""
+    ---------------------------------------------------------------------------
+    OpenCap processing: gaitAnalysis.py
```

Comment:
I suggested to move analyzeGait.py to the Examples folder. I would also move this script to be under ActivityAnalyses and remove the sub-folder ActivityClasses. Open to discuss.